src/Microsoft.TemplateEngine.Core/Util/ProcessorState.cs

```diff
@@ -14,6 +14,8 @@ public class ProcessorState : IProcessorState
     {
         private Stream _source;
         private readonly Stream _target;
+        private byte[] bom;
```

Comment:
for consistency, instance fields seem to be prefixed with an underscore, "_".  We should respect the pattern for both bom & needBom